test/e2e-multicluster-test/multicluster_standalone_test.go

```diff
@@ -130,6 +130,11 @@ var _ = Describe("Test multicluster standalone scenario", func() {
 			_app := &v1beta1.Application{}
 			g.Expect(k8sClient.Get(hubCtx, appKey, _app)).Should(Succeed())
 			_app.Status.Workflow.Suspend = false
+			for i, step := range _app.Status.Workflow.Steps {
+				if step.Type == "suspend" {
+					_app.Status.Workflow.Steps[i].Phase = oamcomm.WorkflowStepPhaseSucceeded
+				}
+			}
```

Comment:
It might be better if we have some function like `ResumeWorkflow(&app.Status.Workflow)`?